fl4health/clients/fenda_client.py

```diff
@@ -108,8 +114,31 @@ def update_before_train(self, current_server_round: int) -> None:
         assert isinstance(self.model, FendaModel)
         if self.perfcl_loss_weights:
             self.aggregated_global_module = self.clone_and_freeze_model(self.model.global_module)
+        if self.mkmmd_loss_weight:
+            self.set_optimized_betas(self.mkmmd_loss)
+
         return super().update_before_train(current_server_round)
 
+    def set_optimized_betas(self, mkmmd_loss: MkMmdLoss) -> None:
+        """Set the optimized betas for the MK-MMD loss."""
+        assert isinstance(self.model, FendaModel)
+
+        local_dist = []
```

Comment:
I'm not quite sure what the suffix `_dist` here is indicating? Perhaps distance? However, these are storing features, so that seems a bit misleading.

Reply:
Distribution, because input for mkmmd loss is distribution